<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>638</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Wazo Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../wazoclient.qrc">
    <normaloff>:/images/wazoicon-color.png</normaloff>:/images/wazoicon-color.png</iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_client">
    <property name="title">
     <string>&amp;Wazo Client</string>
    </property>
    <addaction name="action_configure"/>
    <addaction name="action_to_systray"/>
    <addaction name="action_show_window"/>
    <addaction name="separator"/>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about_client"/>
    <addaction name="action_about_qt"/>
    <addaction name="action_credits"/>
   </widget>
   <addaction name="menu_client"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="action_configure">
   <property name="text">
    <string>Confi&amp;gure</string>
   </property>
   <property name="statusTip">
    <string>Configure account and connection options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="action_to_systray">
   <property name="text">
    <string>To S&amp;ystray</string>
   </property>
   <property name="statusTip">
    <string>Enter the system tray</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="statusTip">
    <string>Connect to the server</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="statusTip">
    <string>Disconnect from the server</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Close the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_show_window">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Show window</string>
   </property>
   <property name="statusTip">
    <string>Leave the system tray</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_about_client">
   <property name="text">
    <string>&amp;About Wazo Client</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_credits">
   <property name="text">
    <string>&amp;Credits</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../wazoclient.qrc"/>
 </resources>
 <connections/>
</ui>
